enclave-runtime/src/ocall/on_chain_ocall.rs

```diff
@@ -16,18 +16,16 @@
 
 */
 
-use crate::{ocall::ffi, rpc::author::alloc::prelude::v1::Vec};
```

Comment:
Good catch, I think that was a CLion auto-import

Reply:
Yeah, might be. But the underlying problem is probably that we reexport it there, which is why Clion even finds that import.